delta/sourcing/src/main/scala/ch/epfl/bluebrain/nexus/sourcing/processor/ShardedAggregate.scala

```diff
@@ -17,12 +17,14 @@ import scala.reflect.ClassTag
   * Relies on cluster sharding to distribute the work for the given
   * (State, Command, Event, Rejection)
   *
+  * @param eventLog      access to the persistent event log
   * @param entityTypeKey the key of an entity type, unique
   * @param clusterSharding the sharding for this aggregate
   * @param retryStrategy the retry strategy to adopt
   * @param askTimeout the ask timeout
   */
 private[processor] class ShardedAggregate[State, Command, Event, Rejection](
+    eventLog: EventLog[Event],
```

Comment:
I was seeing Aggregate and eventLog as two separate things but why not, if it is easier to use that way

Reply:
Ok to add eventLog in there but maybe we can find a more relevant name ?

Reply:
the type is EventLog.... so it seemed appropriate. Do you have a better suggestion?

Reply:
I was thinking more about Aggregate/ShardedAggregate but I have nothing coming in mind so it may be ok :)